src/Integration/packages.config

```diff
@@ -1,7 +1,7 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<packages>
-  <package id="Microsoft.IdentityModel.Clients.ActiveDirectory" version="2.20.301151232" targetFramework="net452" />
-  <package id="Newtonsoft.Json" version="7.0.1" targetFramework="net452" />
-  <package id="NuGet.VisualStudio" version="3.3.0" targetFramework="net452" />
-  <package id="SonarLint" version="1.7.0" targetFramework="net452" developmentDependency="true" />
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.IdentityModel.Clients.ActiveDirectory" version="2.20.301151232" targetFramework="net452" />
+  <package id="Newtonsoft.Json" version="6.0.8" targetFramework="net46" />

```

Comment:
the tool did it. In general 4.6 is shipped with VS and out projects are build against it as well. The question is why the other bits are in 4.52? Don't know and changing that will mean that we need to retest everything again, which we don't want. So if this was working when everything was 4.52 it will also work in 4.6. I'll log a bug on that.